microsoft/testsuites/core/cpu.py

```diff
@@ -0,0 +1,70 @@
+from assertpy.assertpy import assert_that
+from semver import VersionInfo
+
+from lisa import Logger, testsuite
+from lisa.base_tools.cat import Cat
+from lisa.base_tools.uname import Uname
+from lisa.node import Node
+from lisa.tools.lscpu import Lscpu
+from lisa.util import SkippedException
+
+NUMA_OFF = "numa=off"
+MAX_NUMA_UNSUPPORTED_KERNEL = VersionInfo(2, 6, 37)
```

Comment:
1. It's used once, so it doesn't need to defined here.
2. The VersionInfo supports to compare with a string. So the comparison can be like `if version < "2.6.37":`